### ISSUE TITLE
While navigating to the 'Get started/React Native/Windows Design/GitHub...' etc links present in home page, Narrator is not announcing the subtext information

### ISSUE THREAD

**Issue by Nitin-100** [microsoft/react-native-gallery#681]
Build Details:
OS Version: 24H2 
OS Build: 26100.5074
App: React Native Gallery Preview
AT Used: Narrator

Repro-Steps:
Launch the React Native Gallery-Preview app.
Navigate to the 'Expand menu' button and invoke it.
Navigate to ‘Home' button present in left navigation pane and invoke it.
Navigate to the 'Get started/React Native/Windows Design/GitHub...' etc links and listen to the Narrator announcement.
Actual Result:
While navigating to the 'Get started/React Native/Windows Design/GitHub...' etc links present in home page, Narrator is not announcing the subtext information. It is just announcing as "<label name>, <control type>".

Note: Same issue is observed for all the buttons present under the "Recently added samples" & "Recently updated samples" groups  in home page

Expected Result:
Sub text <An overview of app development options, tools, and samples> should be defined in the help text and while navigating to the 'Get started/React Native/Windows Design/GitHub...' etc links present in home page, Narrator should convey the sub text information. Narrator should convey as '<Label name>, <sub text>, <control type> '.

User Impact:
Users who rely on Screen reader will face difficulty in understanding the purpose of the links, if Narrator is not conveying the complete information of the links.

MAS Reference Link: [MAS 4.2.1 – Object Information](https://aka.ms/MAS4.2.1)

Trap ID: [1.4 - Uncomprehended Element](https://aka.ms/UncomprehendedElement)

“Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.”
